crates/hulk_behavior_simulator/build.rs

```diff
@@ -28,6 +28,7 @@ fn main() -> Result<()> {
                     "control::primary_state_filter",
                     "control::motion::look_around",
                     "control::motion::motion_selector",
+                    "control::penalty_shot_direction_estimation",
```

Comment:
You should remove the corresponding main output from the fake data node.
I think there's a bug in our framework where multiple nodes can produce the same main outputs. You just got lucky here that the fake data node was scheduled earlier, so the penalty shot direction estimation node was able to overwrite the data again before it was used.

Reply:
Good point, done